lib/Value/String.pm

```diff
@@ -80,7 +80,16 @@ sub compare {
 #
 #  Mark a string to be display verbatim
 #
-sub verb {shift; return "\\verb".chr(0x1F).(shift).chr(0x1F)}
+sub verb {
+  shift;
+  my $s = shift;
+  $s =~ 's/\r/ /g';
+  return "{\\verb\r$s\r}" unless $s =~ m/\n/;
+  my @tex = split(/\n/,$s);
+  foreach (@tex) {$_ = $self->verb($_) if $_ =~ m/\S/}
```

Comment:
I'm not sure the recursion is really necessary in this case.  The removal of `\r` has already been done, so there is no need to redo that or to perform the setup and teardown for a function call, here.  I would think

```
  foreach (@tex) {$_ = "\\verb\r$s\r" if $_ =~ m/\S/}
```

would be sufficient (also no need for the braces in this case, as the only think following `\verb` will be either `\\` or `\end{array}`).

Reply:
OK, that tracks.